components/raftstore/src/store/metrics.rs

```diff
@@ -714,6 +714,21 @@ lazy_static! {
             "Total number of leader missed region."
         ).unwrap();
 
+    pub static ref READ_INDX_COAL_COUNTER: IntCounter = register_int_counter!(
+        "tikv_readIndex_coal",
```

Comment:
Sorry but you are mixing snake_case and camenCase. Please use snake_case only here.